node/src/syncer.rs

```diff
@@ -46,8 +46,8 @@ pub enum SyncerError {
     Store(#[from] StoreError),
 
     /// An error propagated from the [`celestia_types`].
-    #[error(transparent)]
-    Celestia(#[from] celestia_types::Error),
+    #[error(" Celestia error: {0}")]
+    Celestia(celestia_types::Error),
```

Comment:
This is not needed after you implement `std::error::Error` as explained in the other comment.